.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# http://editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{cpp,c,hpp,h}]
+indent_style = tab
```

Comment:
should be space now?

Reply:
We rather want GitHub to display tabs as 4 spaces, so I'm not sure it will work with `indent_style = spaces`.

Reply:
ok, let's try